.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  Spec:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        check: ['syntax lint metadata_lint check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop']
+        ruby_version: [2.5.x]
+        puppet_gem_version: [~> 6.0]
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby_version }}
+
+    - name: Spec Tests
```

Comment:
This should probably be static checks or something. Bit misleading since it doesn't run any spec tests.